DEVELOP.md

```diff
@@ -1,446 +1,443 @@
-      colormap: "grey"
-    },
-  ]
-  // make an array of meshes to load
-  let meshList = [
-    { url: "https://niivue.github.io/niivue/features/images/BrainMesh_ICBM152.lh.mz3"},
-  ]
-  const nv = new Niivue();
-  nv.attachTo("gl"); // attach to canvas with id="gl"
-  nv.setSliceMM(true) // world space coordinates to be used by all images when loading volumes and meshes together
-  nv.loadVolumes(volumeList); // async
-  nv.loadMeshes(meshList); // async
-  ```
-## Set properties before loading images
-  
-  ```js
-    import { Niivue } from "@niivue/niivue";
-    // make an array of volumes to load
-    let volumeList = [
-      { 
-        url: "https://niivue.github.io/niivue/features/images/mni152.nii.gz",
-        colormap: "grey"
-      },
-    ]
-    const nv = new Niivue({
-      isColorbar: true, // show colorbar
-      isOrientationCube: true, // show orientation cube in bottom corner
-      crosshairColor: [0, 1, 0, 0.5], // set crosshair color to green and 50% opacity
-      show3Dcrosshair: true, // show crosshair in 3D view
-      backColor: [1, 1, 1, 1], // set background color to white
-      crosshairWidth: 4, // make crosshair thicker
-      // for more options see: https://niivue.github.io/niivue/devdocs/global.html#NiivueOptions
-    });
-    nv.attachTo("gl"); // attach to canvas with id="gl"
-    nv.loadVolumes(volumeList); // async
-  ```
-
-### Supported Nodejs versions for building/developing with NiiVue
-
-NiiVue is built and tested using [Nodejs](https://nodejs.dev/en/) `14`, `16`, and `18` and their accompanying npm versions. We recommend using the latest LTS version of Nodejs when building and developing with NiiVue.
-
-### Install NiiVue using npm
-
-Installs without the bloat of the development dependencies or testing frameworks
-
-```
-npm install --only=prod @niivue/niivue
-```
-
-# Framework Usage
-
-NiiVue can be used with your favorite web framework - Vue, Angular, React or even pure HTML.
-
-## Pure HTML
-
-While NiiVue can be wrapped with frameworks (VueJS, React, Angular), you can also embed it directly into HTML web pages. Here is a minimal example:
-
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>NiiVue</title>
-  </head>
-  <body>
-    <canvas id="gl" width="640" height="640"></canvas>
-  </body>
-  <script src="https://niivue.github.io/niivue/features/niivue.umd.js"></script>
-  <script>
-    var volumeList = [
-      { url: "https://niivue.github.io/niivue-demo-images/mni152.nii.gz" },
-    ];
-    var nv = new Niivue({ isResizeCanvas: false });
-    nv.attachTo("gl");
-    nv.loadVolumes(volumeList);
-  </script>
-</html>
-```
-
-Note that the code above will load the latest current stable release. Alternatively, you can specify a specific NiiVue version. For example, to ensure compatibility with release 0.32.0, you could specify:
-
-```html
-<script src="https://unpkg.com/@niivue/niivue@0.32.0/dist/niivue.umd.js"></script>
```

Comment:
I realize we no longer provide fat builds, but this is a convenient feature. Are we sure we do not want to find another mechanism to support this ability.